README.md

```diff
@@ -58,5 +58,16 @@ This can be verified in the tests; tests will not pass with checkstyle errors.
 ## Releases
 The `aws-advanced-wrapper-driver` has a regular monthly release cadence. A new release will occur during the last week of each month. However, if there are no changes since the latest release, then a release will not occur.
 
+## Aurora Engine Version Testing
+This driver is being tested against the following versions in the current test suite
+| Database          | Versions                                                                                                                                                                                                         |
+|-------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| MySQL             | 8.0.32                                                                                                                                                                                                           |
+| PostgreSQL        | 15.2                                                                                                                                                                                                             |
+| Aurora MySQL      | MySQL	8.0.mysql_aurora.3.02.2 (Wire-compatible with MySQL 8.0.23 onwards. For more details see [here](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraMySQLReleaseNotes/AuroraMySQL.Updates.3022.html))                     |
+| Aurora PostgreSQL | 14.5 (Compatible with PostgreSQL 14.5, see release notes [here](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraPostgreSQLReleaseNotes/AuroraPostgreSQL.Updates.html#AuroraPostgreSQL.Updates.20180305.145X)) |
+
+The driver is compatible with MySQL 5 and MySQL 8 as per the Community MySQL JDBC Driver.
```

Comment:
While MySQL support is experimental at the moment, the `aws-advanced-jdbc-wrapper` is compatible with MySQL 5.7 and MySQL 8.0 as per the Community MySQL Connector/J 8.0 Driver.